src/ui/components/CredentialOptions/CredentialOptions.test.tsx

```diff
@@ -0,0 +1,139 @@
+import { fireEvent, render, waitFor } from "@testing-library/react";
+import { Provider } from "react-redux";
+import configureStore from "redux-mock-store";
+import { Store, AnyAction } from "@reduxjs/toolkit";
+import { act } from "react-dom/test-utils";
+import { ionFireEvent, waitForIonicReact } from "@ionic/react-test-utils";
+import { TabsRoutePath } from "../navigation/TabsMenu";
+import { CredentialOptions } from "./CredentialOptions";
+import { credsFixAcdc } from "../../__fixtures__/credsFix";
+import EN_TRANSLATIONS from "../../../locales/en/en.json";
+import { setToastMsg } from "../../../store/reducers/stateCache";
+import { ToastMsgType } from "../../globals/types";
+
+jest.mock("@ionic/react", () => ({
+  ...jest.requireActual("@ionic/react"),
+  IonModal: ({ children }: { children: any }) => children,
+}));
+
+const dispatchMock = jest.fn();
+
+describe("Identifier Options modal", () => {
+  let mockedStore: Store<unknown, AnyAction>;
+  beforeEach(() => {
+    jest.resetAllMocks();
+    const mockStore = configureStore();
+    const initialState = {
+      stateCache: {
+        routes: [TabsRoutePath.IDENTIFIERS],
+        authentication: {
+          loggedIn: true,
+          time: Date.now(),
+          passcodeIsSet: true,
+          passwordIsSet: true,
+        },
+      },
+    };
+    mockedStore = {
+      ...mockStore(initialState),
+      dispatch: dispatchMock,
+    };
+  });
+
+  test("should display the modal", async () => {
+    const setCredOptionsIsOpen = jest.fn();
+    const optionDeleteMock = jest.fn();
+
+    const { getByTestId } = render(
+      <Provider store={mockedStore}>
+        <CredentialOptions
+          optionsIsOpen={true}
+          setOptionsIsOpen={setCredOptionsIsOpen}
+          cardData={credsFixAcdc[0]}
+          credsOptionAction={optionDeleteMock}
+        />
+      </Provider>
+    );
+
+    await waitForIonicReact();
+
+    // expect(getByTestId("creds-options-modal")).toBeVisible();
```

Comment:
We should avoid leaving commented out code, please remove.

The reason why this test id is not showing up is because with that snippet I gave you we are able to render the content inside the modal, but not the modal itself, which is where you will find the test id. In this case checking if the elements inside are being rendered would be enough (just like you did with the two buttons below).